index.js

```diff
@@ -11,13 +11,30 @@ var is = require('unist-util-is')
 
 var dashes = /-([a-z])/g
 
+var vueUtils = {
+  isTopLevel: makeMap([
```

Comment:
Is this really needed? Why not a simple array here, and below instead of `.isTopLevel()`, do `.topLevel.indexOf(name) !== -1`

Reply:
Hi ! i just ported this over from [`babel-plugin-transform-vue-jsx`](https://github.com/vuejs/babel-plugin-transform-vue-jsx/blob/3c6dcca1c879d7882449a44fef2536fd787c7ce9/lib/group-props.js#L2) and thought it was probably for perf. reasons on a huge number of nodes (object prop access vs. array indexof).
I didn't measure this though, do you think that would be warranted, or simply replace this by your suggestion ?

cf. https://stackoverflow.com/questions/32234733/javascript-what-lookup-is-faster-array-indexof-vs-object-hash
Since the set size is really small, it probably makes sense to make this simpler to read.